examples/kotlinExample/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.1.1'
+    ext.kotlin_version = '1.1.2'
```

Comment:
The latest version is actually `1.1.2-2` (https://github.com/jetbrains/kotlin#readme)